builtin/providers/google/resource_compute_instance.go

```diff
@@ -58,6 +58,12 @@ func resourceComputeInstance() *schema.Resource {
 							ForceNew: true,
 						},
 
+						"nvme": &schema.Schema{
```

Comment:
Instead of a boolean, consider making this a string for the interface name. That way if there's ever a third option the code will just work.